### ISSUE TITLE
Replace 'add to my inventaire' to more explicti option

### ISSUE THREAD

**Issue by jum-s** [inventaire/inventaire-client#130]
As a user, when i arrive for the first time on inventaire and i dont read the instructions/documentations.
I want to search for editions available to me aka. as ive no friends yet, `public-items`

**Comment by jum-s** [comment]
bonus The button label "Add to my inventaire" can be confusing for newcomers. Expliciting the intention with "propose an edition"  or "propose a work" might ease the experience. 
Confusion might also happen because the concept of "inventaire" might not be understood yet. 
user feedback : "what is inventaire in "add to my inventaire" ? it is the app name and now they want me to add an edition to my inventaire ?"

**Comment by maxlath** [comment]
Actually, the (2) is already there, in the sense that if someone has an item linked to the current edition or work, it will appear below the "add to my inventory" button if they are in my network or share the book publicly ([example](https://inventaire.io/entity/isbn:9782290349229?lang=en)). The issue, is that if no one has anything, nothing appears: that should at least display something like "no one has it".

But yes, the actions that can be taken from a work or an edition need to be rethinked, in my opinion, in combination with the ["I'm looking for this book"](https://trello.com/c/S5wGWC78/55-im-looking-for-this-book-mode-wishlist), and ["I want to write a review of this book"](https://trello.com/c/uwdkvGl1/114-book-reviews) features.

**Comment by thewholelifetolearn** [comment]
There is also another confusion because even if the edition is in my library, the button "add to my inventaire" is displayed.  
If the purpose is to add to the library, the button shouldn't be displayed. Or display in another way in the case someone has the same edition multiple times.

**Comment by jum-s** [comment]
as a first time user who is not expert in social networks, the concept of "inventory" itself is not explained enough, with available inventaires, what can be done ? Intentions could be more explicit.

"I have this book"
"I can share this book"
"I want this book"

**Comment by thewholelifetolearn** [comment]
I agree. You gave a good example. There is a need to be "less expert".
